src/screens/splash/Splash.js

```diff
@@ -0,0 +1,59 @@
+import {Image, StyleSheet, Text, View} from 'react-native';
+import React, {useEffect} from 'react';
+import route from '../../utils/constants/routeConstants';
+const bookImage = require('../../assets/images/book.png');
+
+const Splash = ({navigation}) => {
+  useEffect(() => {
+    const timer = setTimeout(() => {
+      navigation.replace(route.LOGIN);
+    }, 5000);
+    return () => clearTimeout(timer);
```

Comment:
We don't need to clear timeout as we are not calling it again from anywhere else